bundles/org.eclipse.swt/Eclipse SWT/win32/org/eclipse/swt/graphics/FontMetrics.java

```diff
@@ -208,10 +215,34 @@ public int hashCode() {
  *
  * @noreference This method is not intended to be referenced by clients.
  */
+@Deprecated
```

Comment:
```suggestion
@Deprecated(forRemoval = true, since = "2024-09")
```

Reply:
There is no need to deprecate methods marked with noreference as it's platform specific and thus not part of SWT API, feel free to remove it directly.

Reply:
Done

Reply:
@akurtakov would it be wrong to mark it as deprecated for removal? Please don't get me wrong, I prefer to remove it too but deprecating was also an option (https://github.com/eclipse-platform/eclipse.platform.swt/pull/1243/files#r1609580640)

Reply:
This method doesn't exist in Cocoa and Gtk port thus deprecating it will not buy you/us anything. While technically not wrong to mark unused "private"(implementation specific) method it just leaves us with garbage in the codebase that too often people forget to collect later.

Reply:
It's worth mentioning that javadoc published in doc bundles and help center are built on Linux thus this deprecation message and notification will not end up in "user visible" location.

Reply:
Ok, so I guess it's no use to deprecate it. Thank you for clarifying this! @amartya4256 you may remove the deprecated method and push your changes :)

Reply:
@akurtakov fyi we decided to remove the deprecation notice and leave the method because the only remaining usage is going to be removed in #1244 anyway and the method will be removed there.